### DISCUSSION TITLE
vosk models (not updating to new version)

### DISCUSSION THREAD

**Discussion by afghklhgfjdsg** [SubtitleEdit/subtitleedit#7104] [General]
The list of vosk models do not show current versions. I have to download them for the current version.
https://alphacephei.com/vosk/models

For example when it downloads spanish this is downloaded "vosk-model-small-es-0.3". Current versions are vosk-model-small-es-0.42 & vosk-model-es-0.42

I should add I am on on subtitle edit 3.6.13. I deleted the vosk folder and redownloaded new with same thing. I am fine downloading from site as easy enough, but people may not be aware they are using old/bad models.

**Comment by niksedk** [comment]
Thx for the info :)

SE source file is here:
https://github.com/SubtitleEdit/subtitleedit/blob/main/src/libse/AudioToText/VoskModel.cs

**Comment by niksedk** [comment]
Updated somewhat here: https://github.com/SubtitleEdit/subtitleedit/commit/ff8df4d260c0878c66c75607410bc36a54a803d7
